services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/events.py

```diff
@@ -250,6 +230,16 @@ async def action(cls, app: FastAPI, scheduler_data: SchedulerData) -> None:
                 dynamic_sidecar_endpoint
             )
         except BaseClientHTTPError:
+            were_service_containers_detected_before = (
+                len(scheduler_data.dynamic_sidecar.containers_inspect) > 0
+            )
+            if were_service_containers_detected_before:
+                # Containers disappeared after they were started.
```

Comment:
I think here we need to discuss that part I do not see it.

and also the warning just below.

Reply:
As discussed this covers some edge cases to avoid manual cleanup